### ISSUE TITLE
About the datePicker.datePickerMode

### ISSUE THREAD

**Issue by liuyan3176** [DavdRoman/Bohr#27]
Hi David,
Thanks for your Bohr,it help me a lot in my project. 
But now I have to change the datePicker.datePickerMode=UIDatePickerModeDateAndTime,
When I scroll the datePicker at today,the project will crash and the log is in the following: 
**\* Terminating app due to uncaught exception 'CALayerInvalidGeometry', reason: 'CALayer position contains NaN: [nan nan]' ***
Can you help me and tell me how to solve this problem?
Thanks a lot.

**Comment by davdroman** [comment]
Hi there!

Thanks for reporting that! It's a known issue and I'll try to address it in the future 4.0 release. I'm pretty sure it has to do with the way cells calculate their height based on their elements (`UIPickerView` is a super heavy element layout-wise, especially in `DateAndTime` mode).

**Comment by liuyan3176** [comment]
@DavdRoman  Thanks a lot !

**Comment by Steven4294** [comment]
any luck fixing this?

**Comment by davdroman** [comment]
@Steven4294 not really, I haven't been able to work on Bohr anymore since I'm quite busy. However I encourage you to look into it and PR a fix for this if you'd like 😄
